README.md

```diff
@@ -1,2 +1,105 @@
 # PackIt
-A Link Shortener
+Welcome to the PackIt project! PackIt is a Link Shortener application build with mordern tech stack.
+
+
+### Prerequisites
+Make sure you have the following installed:
+- [Java 17+](https://adoptopenjdk.net/): Programming Language and runtime environment.
```

Comment:
We are using kotlin programmin\ng language instead of java and even though kotlin runs on top of  JVM, we still need kotlin itself.
The graal compiler is also a prerequisite

Reply:
Done